src/main/java/ch/njol/skript/expressions/ExprTimeLived.java

```diff
@@ -0,0 +1,77 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.util.Timespan;
+import ch.njol.skript.util.Timespan.TimePeriod;
+import ch.njol.util.Math2;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Time Lived of Entity")
+@Description("""
+	Returns the total amount of time the entity has lived.
+	Note: This does not reset when a player dies.
+	""")
+@Example("clear all entities where [input's time lived > 1 hour]")
+@Example("""
+	on right click on entity:
+		send "%entity% has lived for %time lived of entity%" to player
+	""")
+@Since("INSERT VERSION")
+public class ExprTimeLived extends SimplePropertyExpression<Entity, Timespan> {
+
+	static {
+		register(ExprTimeLived.class, Timespan.class, "time (alive|lived)", "entities");
+	}
+
+	@Override
+	public @Nullable Timespan convert(Entity entity) {
+		return new Timespan(TimePeriod.TICK, entity.getTicksLived());
+	}
+
+	@Override
+	public Class<?> @Nullable [] acceptChange(ChangeMode mode) {
+		return switch (mode) {
+			case ADD, REMOVE, SET, RESET -> CollectionUtils.array(Timespan.class);
+			default -> null;
+		};
+	}
+
+	@Override
+	public void change(Event event, Object @Nullable [] delta, ChangeMode mode) {
+		int newTicks = 1;
```

Comment:
It might be better to store this and `valueToSet` as longs to account for overflows. That is, if the value is set to high for an int, clamp it down to the max int value.

Reply:
hi @MrScopes do you intend to make this change or should I do it for you? (or do you have a disagreement with the suggested change?)

Reply:
> hi @MrScopes do you intend to make this change or should I do it for you? (or do you have a disagreement with the suggested change?)

Hey sorry been super busy and forgot about this. Feel free to do whatever and make whatever changes